IlmBase/Imath/ImathEuler.h

```diff
@@ -226,6 +226,12 @@ class Euler : public Vec3<T>
     Euler(const Matrix33<T> &, Order o = Default);
     Euler(const Matrix44<T> &, Order o = Default);
 
+    //-------------
+    //  Destructor
+    //-------------
+	
+	~Euler();
```

Comment:
We should be able to declare this as ~Euler() = default; and not have to provide the definition below. That should force it to be the empty function - same behavior but fewer lines of code, and the intention is a bit clearer. Same with the other classes below.

Reply:
Sounds good, I will change that everywhere!

Reply:
I pushed a new version, let me know if all looks good now!